zeq/detail/exit.fbs

```diff
@@ -0,0 +1,6 @@
+namespace zeq.vocabulary;
+
+table Exit {
+}
+
+root_type Exit;
```

Comment:
I remember seeing this somewhere else... Where does this one come from, or why is it added now?